scripts/any/8fef1e9/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: generic
+sudo: false
```

Comment:
I was fuzzy. Turns out that to avoid jobs being automatically added for both osx and linux we need to define a matrix in the packages' travis.yml like:

```yml
matrix:
  include:
    - os: linux
      sudo: false
```